libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -277,6 +277,36 @@ export interface IBucketClient {
    * @inflight
    */
   delete(key: string, opts?: BucketDeleteOptions): Promise<void>;
+
+  /**
+   * Tries to put an object in the bucket.
+   * @param key Key of the object.
+   * @param body Content of the object we want to store into the bucket.
+   * @returns `true` if the object was successfully stored, `false` if an error occurred.
+   */
+  tryPut(key: string, body: string): Promise<boolean>;
+
+  /**
+   * Tries to put a Json object in the bucket.
+   * @param key Key of the object.
+   * @param body Json object that we want to store into the bucket.
+   * @returns `true` if the object was successfully stored, `false` if an error occurred.
+   */
+  tryPutJson(key: string, body: Json): Promise<boolean>;
+
+  /**
+   * Tries to retrieve an object from the bucket.
+   * @param key Key of the object.
+   * @returns `true` if the object was successfully retrieved, `false` if an error occurred.
+   */
+  tryGet(key: string): Promise<boolean>;
+
+  /**
+   * Tries to retrieve a Json object from the bucket.
+   * @param key Key of the object.
+   * @returns `true` if the object was successfully retrieved, `false` if an error occurred.
+   */
+  tryGetJson(key: string): Promise<boolean>;
```

Comment:
```suggestion
  /**
   * Gets an object from the bucket if it exists, parsing it as Json.
   * @param key Key of the object.
   * @returns the contents of the object as Json if it exists, nil otherwise
   */
  tryGetJson(key: string): Promise<Json | undefined>;
```